.github/workflows/all_plugins.yaml

```diff
@@ -84,23 +81,27 @@ jobs:
             find . -maxdepth 12 -name "*.java" -print0 \| xargs -0 java -jar $HOME/google-java-format.jar --replace
           ./.github/workflows/scripts/validate-formatting.sh
 
-  # build_examples_dart:
-  #   timeout-minutes: 15
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #     - uses: actions/checkout@v1
-  #       with:
-  #         fetch-depth: 0
-  #     - name: "Install Flutter"
-  #       run: ./.github/workflows/scripts/install-flutter.sh dev
-  #     - name: "Install Tools"
-  #       run: ./.github/workflows/scripts/install-tools.sh
-  #     - name: "Bootstrap Workspace"
-  #       run: melos bootstrap
-  #     - name: "Build Example Snapshots"
-  #       run: |
-  #         melos exec -c 1 --scope="*example*" -- \
-  #           "flutter --suppress-analytics build apk"
+  build_examples_dart:
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 0
+      - name: "Install Flutter"
+        run: ./.github/workflows/scripts/install-flutter.sh beta
+      - name: "Install Tools"
+        run: |
+          ./.github/workflows/scripts/install-tools.sh
+          flutter config --enable-web
+      - name: "Bootstrap Workspace"
+        run: melos bootstrap
+      # TODO This only builds examples that have a web implementation. Previously this was `aot` snapshots however this
+      # TODO has since been removed from Flutter.
+      - name: "Build Examples"
+        run: |
+          melos exec -c 1 --scope="*example*" --dir-exists="web" -- \
```

Comment:
Won't we still risk a timeout?

Reply:
Not for web building, takes about 4 minutes to build all current web apps (7 currently) - current timeout is set to 15 minutes and max GH action timeout is 60 minutes (something we'd easily reach with 5-10 minute each Android builds - but not for web)